src/github_actions.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
 github_actions::get_pr_number() {
-  jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH"
+  local -r pull_request_number=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
+
+  if [[ "$pull_request_number" != "null" ]]; then
+    echo "$pull_request_number"
+  else
+    echo "Not a pull request event"
```

Comment:
this code block would've saved me so much time while I was testing this. I was testing on push events and was getting very obscure errors. 🙃